networkx/algorithms/dag.py

```diff
@@ -56,8 +56,13 @@ def descendants(G, source):
     Examples
     --------
     >>> DG = nx.path_graph(5, create_using=nx.DiGraph)
-    >>> sorted(list(nx.descendants(DG, 2)))
-    [3, 4]
+    >>> descendants = nx.descendants(DG, 2)
+    >>> descendants
+    {3, 4}
```

Comment:
We'll need to keep the `sorted` as sets are not ordered, so giving the set output directly might result in intermittent doctest failures. See [the doctest docs on this subject](https://docs.python.org/3/library/doctest.html#warnings) for more info.

```suggestion
    >>> descendants = nx.descendants(DG, 2)
    >>> descendants
    {3, 4}
```
```suggestion
    >>> sorted(nx.descendants(DG, 2))
    [3, 4]
```